src/Microsoft.OData.Core/ODataBatchReader.cs

```diff
@@ -108,12 +95,54 @@ public ODataBatchReaderState State
                 return this.batchReaderState;
             }
 
-            private set
+            protected set
             {
                 this.batchReaderState = value;
             }
         }
 
+        /// <summary>
+        /// >The batch-specific URL converter that stores the content IDs found in a changeset and supports resolving cross-referencing URLs.

```

Comment:
>> [](start = 12, length = 1)

Nit: remove ">"

Reply:
fixed.

---
In reply to: [140841481](https://github.com/OData/odata.net/pull/960#discussion_r140841481) [](ancestors = 140841481)